keyboards/mechwild/trisk/keymaps/default/keymap.c

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 Kyle McCreery (@kylemccreery)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _BASE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT(
+        KC_MUTE, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, RGB_RMOD, RGB_MOD, RGB_TOG
+    )
+};
+
+#ifdef RGB_MATRIX_ENABLE
+bool rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
+  HSV hsv = {190, 255, 128};
+  RGB rgb = hsv_to_rgb(hsv);
+  for (uint8_t i = led_min; i <= led_max; i++) {
+    if (g_led_config.flags[i] & LED_FLAG_UNDERGLOW) {
+      rgb_matrix_set_color(i, rgb.r, rgb.g, rgb.b);
+    }
+  }
+  return false;
+}
+#endif
```

Comment:
This could/should be moved to the keyboard's c file, ad as the _kb function. 

Also, 
```c
    if (hsv.v > rgb_matrix_get_val()) {
        hsv.v = rgb_matrix_get_val();
    }
```
This will prevent the indicator from being brighter than the rest of the keyboard's.